pyfolio/tears.py

```diff
@@ -76,8 +77,8 @@ def create_returns_tear_sheet(returns, live_start_date=None,
         benchmark_rets = utils.get_symbol_rets('SPY')
         # If the strategy's history is longer than the benchmark's, limit
         # strategy
-        benchmark_start = benchmark_rets.index[0].astimezone(pytz.UTC)
-        returns_start = returns.index[0].astimezone(pytz.UTC)
+        benchmark_start = get_utc_date(benchmark_rets.index[0])
```

Comment:
I'm not sure, I did not alter the original functionality, just ensured time zone conversion would be robust. 

My hunch is that we should do input checking and ensure UTC time stamps for the indexes. 

Sent from my iPhone

> On Jul 29, 2015, at 3:05 AM, Thomas Wiecki <notifications@github.com> wrote:
> 
> In pyfolio/tears.py:
> 
> > @@ -76,8 +77,8 @@ def create_returns_tear_sheet(returns, live_start_date=None,
> >          benchmark_rets = utils.get_symbol_rets('SPY')
> >          # If the strategy's history is longer than the benchmark's, limit
> >          # strategy
> > -        benchmark_start = benchmark_rets.index[0].astimezone(pytz.UTC)
> > -        returns_start = returns.index[0].astimezone(pytz.UTC)
> > +        benchmark_start = get_utc_date(benchmark_rets.index[0])
> Why are we only converting the start and end dates rather then the whole index?
> 
> —
> Reply to this email directly or view it on GitHub.
>